torntools/scripts/content/properties/ttProperties.js

```diff
@@ -0,0 +1,16 @@
+requireDatabase().then(() => {
+	console.log("TT - Properties");
+	let observer = new MutationObserver(addPropertyValues);
+	observer.observe(doc.find("div#properties-page-wrap"), {childList: true});
+	addPropertyValues();
+});
+
+function addPropertyValues() {
+	let propertyValue = null;
+	for (let property of doc.find(".properties-list").children) {

```

Comment:
I would personally change this to `doc.find(".properties-list > *")`, to be consistent in our code.